cpp/leaky_stdlib.cc

```diff
@@ -111,6 +112,10 @@ void execve(Str* argv0, List<Str*>* argv, Dict<Str*, Str*>* environ) {
   InvalidCodePath();
 }
 
+void kill(int pid, int sig) {
+  assert(::kill(pid, sig) == 0);
```

Comment:
To follow Python, this should check the return code and then

```
 throw Alloc<OSError>(errno);
```

Some of the other functions do this, and we should clean up ones that don't


(Annoying thing: Python 2 sometimes throws IOError and sometimes OSError, and it's not documented.   So I had to look at pyext/posixmodule.c  and read posix_error() to see which one it was! )

Reply:
Ah nice. This is less aggressive than assert-failing here. Added